jetty-websocket/jetty-websocket-client/src/main/java/org/eclipse/jetty/websocket/client/WebSocketClient.java

```diff
@@ -18,6 +18,18 @@
 
 package org.eclipse.jetty.websocket.client;
 
+import java.io.IOException;
+import java.net.CookieStore;
+import java.net.SocketAddress;
+import java.net.URI;
+import java.time.Duration;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import java.util.concurrent.ThreadLocalRandom;
```

Comment:
Are you sure your IDE settings are correct for this ordering?  We seam to be fighting back and forth?

Reply:
it's probably me that is wrong...